plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -84,6 +249,57 @@ private function is_latest_woocommerce_version() {
 		return version_compare( WC()->version, $latest_wc_version, '>=' );
 	}
 
+	/**
+	 * Check if the error exclusively contains third-party stack frames for fatal-errors source context.
+	 *
+	 * @param string $message The error message.
+	 * @param array  $context The error context.
+	 *
+	 * @return bool
+	 */
+	protected function is_third_party_error( $message, $context ) {
```

Comment:
*Optional* thing to consider: we could add type-hinting.

```suggestion
	protected function is_third_party_error( string $message, array $context ): bool {
```

Though I realize we'd probably also need a cast further up where this method is called (and so, if we're deliberately being loose, that's probably fine).

Reply:
Sounds good. Added type-hinting and updated code where this method is called in 628fcf690ee0df05679a461d2fdf8ecf10a5335c